plugin/action/mask/mask.go

```diff
@@ -255,8 +244,9 @@ func (p *Plugin) Do(event *pipeline.Event) pipeline.ActionResult {
 		}
 		v.MutateToString(string(p.maskBuf))
 	}
-	if p.logMaskAppeared && maskApplied {
```

Comment:
Same as above: `p.logMaskAppeared` describes does mask appearance needs to be sent by Prometheus.